src/pyodide/internal/pyodide-bootstrap.js

```diff
@@ -0,0 +1,76 @@
+import { loadPyodide } from "pyodide:python";
+import { getMetadata } from "pyodide:current-bundle";
```

Comment:
Where does `pyodide:current-bundle` come from? Does this import just fail currently?

Reply:
It's injected together with this script during WorkerBundle processing (not implemented in workerd yet)